libkatana_python_native/src/EntityTypeManager.cpp

```diff
@@ -103,5 +120,42 @@ katana::python::InitEntityTypeManager(py::module_& m) {
             } else {
               return py::cast(EntityType{self, id});
             }
-          });
+          })
+      .def(
+          "get_atomic_subtypes",
+          [](const katana::EntityTypeManager* self, EntityType type) {
+            py::set ret;
+            if (self->HasEntityType(type.type_id)) {
+              auto type_set = self->GetAtomicSubtypes(type.type_id);
+              for (EntityTypeID subtype_id = 0; subtype_id < type_set.size();
+                   ++subtype_id) {
```

Comment:
This isn't very efficient. It iterates over every bit in the vector, not just the ones. I don't think we should fix it now, but please add a TODO and file an issue against `DynamicBitSet` to add a method that efficiently finds ones (by iterating over 64-bit words and doing bit magic). That would help a lot with this and should be reasonably fast.